### DISCUSSION TITLE
what is   ` "ios:text-blue-500": 8192,` ?

### DISCUSSION THREAD

**Discussion by arnaudambro** [nativewind/nativewind#320] [General]
It's not a bug but there is something I don't get in the docs

Here -> https://github.com/marklawlor/nativewind/blob/main/apps/website/docs/overview/how-it-works.md#state

```
<Text class="text-black ios:text-blue-500" />;

NativeWindStyleSheet.create({
  styles: {
    "text-black": {
      color: "#000",
    },
    "ios:text-blue-500": {
      width: "rgb(59 130 246)",
    },
  },
  masks: {
    "ios:text-blue-500": 8192,
  },
});
```

I don't get the value `8192`, what is it ?

**Comment by marklawlor** [comment]
A bitmask value that was used in v2. The `next` version will remove this entirely #308

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
